pkgs/by-name/wi/winchecksec/package.nix

```
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pe-parse,
+  uthenticode,
+  openssl,
+  gtest,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "winchecksec";
+  version = "3.1.0";
+
+  src = fetchFromGitHub {
+    owner = "trailofbits";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-KK0mQiPY/LNCcY4z1smpRjzyLhVcpm/k2ReCUB2hq1Y=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    openssl
+  ];
+  nativeCheckInputs = [ gtest ];
+
+  buildInputs = [
+    pe-parse
+    uthenticode
+  ];
+
+  patchPhase = ''
+    substituteInPlace CMakeLists.txt \
+      --replace 'if (BUILD_TESTS)' $'find_package(GTest CONFIG REQUIRED)\nadd_subdirectory(test)\nif (FALSE)' \
+  '';
+
+  doCheck = true;
+  checkPhase = "test/winchecksec_test";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    install -Dm555 winchecksec $out/bin/
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
- https://github.com/NixOS/nixpkgs/issues/371862